tools/v8_gypfiles/v8.gyp

```diff
@@ -1378,6 +1378,9 @@
           '<(V8_ROOT)/src/heap/base/worklist.h',
         ],
         'conditions': [
+          ['enable_lto=="true"', {
+            'cflags_cc': [ '-fno-lto' ],
```

Comment:
This could be set regardless of the `enable_lto` setting?

Reply:
Technically, yes, at least for GCC. But I am not sure about MSVC or Clang. Since `enable_lto` implies some other conditions, I think it is better to have this conditional.